app/src/NativeBase/Components/Forms/DateTime.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file DatePicker.
```

Comment:
As this component only allows choice of date not time, could you rename it to something like `DatePicker` like your description?